app/controllers/invite_controller.rb

```diff
@@ -20,6 +20,15 @@ def submit
     first_name = params[:first_name]
     last_name = params[:last_name]
 
+    if ENV["RESTRICTED_DOMAIN"]
+      if email.split("@").last != ENV["RESTRICTED_DOMAIN"]
+        @message = "Sorry! Early access is currently restricted to people within the #{ENV["RESTRICTED_DOMAIN"]} domain.  We will be opening it up soon, though!"
```

Comment:
@KrauseFx Oops -- you're right.  This was copied from my fork verbatim. I'll change that.